vllm/core/scheduler.py

```diff
@@ -136,6 +121,7 @@ def _schedule(self) -> SchedulerOutputs:
             # Optimization: We do not sort the waiting queue since the preempted
             # sequence groups are added to the front and the new sequence groups
             # are added to the back.
+            self.waiting = self.policy.sort_by_priority(self.waiting)
```

Comment:
Could you add the `if` check to avoid sorting when the policy is `fcfs`? Please also move the Optimization comment to the `if` check.